// 404-page/dist/index.modern.js
import React from 'react';
import image from './notfound~bfvlKBqd.png';
import imageWithoutFace from './notfound2~xBRDZkYd.png';
import space from './space-bg~DEmLjkSk.jpg';

var styles = {"mainImg":"_1Z8Jv","spin":"_1sp4O","main":"_S55cW","title":"_2KezC","lost":"_fXlyq","title2":"_29wBp","face":"_33ZtY","images":"_MSO0O"};

var LostPage = function LostPage(_ref) {
  var backgroundColor = _ref.backgroundColor,
      face = _ref.face,
      audio = _ref.audio;

  var checkBg = function checkBg() {
    switch (backgroundColor) {
      case "purple":
        return {
          backgroundColor: "#595357",
          backgroundImage: "linear-gradient(225deg, #595357 0%, #242225 50%, #404548 100%)"
        };

      case "black":
        return {
          backgroundColor: "#595357",
          backgroundImage: "linear-gradient(225deg, #595357 0%, #242225 50%, #404548 100%)"
        };

      case "purple":
        return {
          backgroundColor: "A9C9FF",
          backgroundImage: "linear-gradient(180deg, #A9C9FF 0%, #FFBBEC 100%)"
        };

      case "blue":
        return {
          backgroundColor: "#0093E9",
          backgroundImage: "linear-gradient(160deg, #0093E9 0%, #80D0C7 100%)"
        };

      case "red":
        return {
          backgroundColor: "red"
        };

      default:
        return {
          backgroundImage: "url(" + space + ")"
        };
    }
  };

  return /*#__PURE__*/React.createElement("div", {
    className: styles.main,
    style: checkBg()
  }, /*#__PURE__*/React.createElement("h1", null, "\xA0\xA0\xA0\xA0"), /*#__PURE__*/React.createElement("h1", {
    className: styles.title
  }, "you are"), /*#__PURE__*/React.createElement("h1", {
    className: styles.title2
  }, "so lost"), audio && /*#__PURE__*/React.createElement("audio", {
    src: audio,
    autoPlay: true,
    type: "audio/mp3"
  }), face ? /*#__PURE__*/React.createElement("div", {
    className: styles.images
  }, /*#__PURE__*/React.createElement("img", {
    src: imageWithoutFace,
    className: styles.mainImg
  }), /*#__PURE__*/React.createElement("img", {
    src: face,
    className: styles.face
  })) : /*#__PURE__*/React.createElement("img", {
    src: image,
    className: styles.mainImg
  }));
};

export { LostPage };
